tests/wolfssl-test/main.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2019 Kaleb J. Himes
+ *
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       wolfSSL cryptographic library test
+ *
+ * @author      Kaleb J. Himes <kaleb@wolfssl.com>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include <wolfssl/wolfcrypt/settings.h>
+#include <wolfcrypt/test/test.h>
+#ifdef MODULE_WOLFCRYPT_BENCHMARK
+#include <wolfcrypt/benchmark/benchmark.h>
+#endif
+
+int main(void)
+{
+    puts("wolfSSL Crypto Test!");
+    wolfcrypt_test(NULL);
```

Comment:
please add `xtimer_sleep(1);` before calling `wolfcrypt_test` (and the required `#include "xtimer.h"` to the top) to workaround a failing test on platforms that don't have a synced RTC, as I explained in [one of my previous comments](https://github.com/RIOT-OS/RIOT/pull/10308#issuecomment-519430658).